### PR TITLE
Fix ptf log and pcap file can't be fetch issue

### PR DESCRIPTION
**Pull request by xwjiang-ms**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
1. ptf collect cannot fetch log and pcap file because it excute before
log file generate
2. new log and pcap files will cover old files if filenames are the same
defined in testcases
#### How did you do it?
1. fetch after log and pcap files generate whether test fails or not
2. add timestamp to filename in docker, and files in ptf will not be
changed
#### How did you verify/test it?
Run a test which would call ptf runner

### CODE REVIEW

File: tests/ptf_runner.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
 logger = logging.getLogger(__name__)
 
 
 def ptf_collect(host, log_file):
-    host.fetch(src=log_file, dest='./logs/ptf_collect/', flat=True, fail_on_missing=False)
     pos = log_file.rfind('.')
-    pcap_file = (log_file[0:pos] if pos > -1 else log_file) + '.pcap'
+    filename_prefix = log_file[0:pos] if pos > -1 else log_file
+    rename_prefix = filename_prefix.replace('/tmp/', '')
```
Comment (ZhaohuiS) [inline tests/ptf_runner.py:12]: Why do we need tmp folder here?

Comment (ZhaohuiS) [inline tests/ptf_runner.py:12]: Are all log files saved to /tmp/ folder？Is it better to split folder name and file name with "/"? Dest only cares file name, then we don't have to replace /tmp here.

Comment (xwjiang-ms) [reply tests/ptf_runner.py]: Yes, we don't have to replace /tmp/

File: tests/ptf_runner.py [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 
 logger = logging.getLogger(__name__)
 
 
 def ptf_collect(host, log_file):
-    host.fetch(src=log_file, dest='./logs/ptf_collect/', flat=True, fail_on_missing=False)
     pos = log_file.rfind('.')
-    pcap_file = (log_file[0:pos] if pos > -1 else log_file) + '.pcap'
+    filename_prefix = log_file[0:pos] if pos > -1 else log_file
+
```
Comment (ZhaohuiS) [inline tests/ptf_runner.py:12]: For this change, it seems to carry the folder name as well, which is not reasonable.
My original opinion is if log file looks like "`/tmp/somecase-logfile.log`", then just use `somecase-logfile` as prefix.
If log file has multiple folders, looks like this `/tmp/subfolder/somecase-logfile.log`, then just use `somecase-logfile.log` as prefix too.

Comment (xwjiang-ms) [reply tests/ptf_runner.py:12]: Yeah, make sense

**Review by ZhaohuiS — APPROVED** [review]

**Pull request merged**